src/commandSocket.cpp

```diff
@@ -1,472 +1,17 @@
-//     socket.sendInt32(extruder_nr);
-//     socket.sendFloat32(print_time);
-}
-
-void CommandSocket::beginSendSlicedObject()
-{
-#ifdef ARCUS
-    if(!private_data->sliced_object_list)
-    {
-        private_data->sliced_object_list = std::make_shared<cura::proto::SlicedObjectList>();
-    }
-
-    private_data->current_sliced_object = private_data->sliced_object_list->add_objects();
-    private_data->current_sliced_object->set_id(private_data->object_ids[private_data->sliced_objects]);
-#endif
-}
-
-void CommandSocket::endSendSlicedObject()
-{
-#ifdef ARCUS
-    private_data->sliced_objects++;
-    private_data->current_layer_offset = private_data->current_layer_count;
-    std::cout << "End sliced object called. Sliced objects " << private_data->sliced_objects << " object count: " << private_data->object_count << std::endl;
-
-    if(private_data->sliced_objects >= private_data->object_count)
-    {
-        private_data->socket->sendMessage(private_data->sliced_object_list);
-        private_data->sliced_objects = 0;
-        private_data->current_layer_count = 0;
-        private_data->current_layer_offset = 0;
-        private_data->sliced_object_list.reset();
-        private_data->current_sliced_object = nullptr;
-        auto done_message = std::make_shared<cura::proto::SlicingFinished>();
-        private_data->socket->sendMessage(done_message);
-    }
-#endif
-}
-
-void CommandSocket::sendFinishedSlicing()
-{
-#ifdef ARCUS
-    std::shared_ptr<cura::proto::SlicingFinished> done_message = std::make_shared<cura::proto::SlicingFinished>();
-    private_data->socket->sendMessage(done_message);
-#endif
-}
-
-void CommandSocket::beginGCode()
-{
-#ifdef ARCUS
-    FffProcessor::getInstance()->setTargetStream(&private_data->gcode_output_stream);
-#endif
-}
-
-void CommandSocket::flushGcode()
-{
-#ifdef ARCUS
-    auto message = std::make_shared<cura::proto::GCodeLayer>();
-    message->set_id(private_data->object_ids[0]);
-    message->set_data(private_data->gcode_output_stream.str());
-    private_data->socket->sendMessage(message);
-    
-    private_data->gcode_output_stream.str("");
-#endif
-}
-
-void CommandSocket::sendGCodePrefix(std::string prefix)
-{
-#ifdef ARCUS
-    auto message = std::make_shared<cura::proto::GCodePrefix>();
-    message->set_data(prefix);
-    private_data->socket->sendMessage(message);
-#endif
-}
-
-#ifdef ARCUS
-cura::proto::Layer* CommandSocket::Private::getLayerById(int id)
-{
-    id += current_layer_offset;
-
-    auto itr = std::find_if(current_sliced_object->mutable_layers()->begin(), current_sliced_object->mutable_layers()->end(), [id](cura::proto::Layer& l) { return l.id() == id; });
-
-    cura::proto::Layer* layer = nullptr;
-    if(itr != current_sliced_object->mutable_layers()->end())
-    {
-        layer = &(*itr);
-    }
-    else
-    {
-        layer = current_sliced_object->add_layers();
-        layer->set_id(id);
-        current_layer_count++;
-    }
-
-    return layer;
-}
-#endif
```

Comment:
It's a shame git didn't see this commit as a rename of commandSocket.cpp to commandSocketArcus.cpp ....